src/internal/hooks/use-mobile/index.ts

```diff
@@ -4,7 +4,15 @@ import { createSingletonState } from '@cloudscape-design/component-toolkit/inter
 
 import { getMatchingBreakpoint, mobileBreakpoint } from '../../breakpoints';
 
+export const forceMobileModeSymbol = Symbol.for('awsui-force-mobile-mode');
+
 function getIsMobile() {
+  // allow overriding the mobile mode in tests
+  // any is needed because of this https://github.com/microsoft/TypeScript/issues/36813
+  const forceMobileMode = (globalThis as any)[forceMobileModeSymbol];
+  if (typeof forceMobileMode !== 'undefined') {
+    return forceMobileMode;
+  }
```

Comment:
This follows our VR flags idea: https://github.com/cloudscape-design/component-toolkit/pull/79

Currently this is used only internally, but we might make it our official test util to allow our consumers write unit tests on mobile use-cases